plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -448,7 +449,9 @@ private List<TableInfo> getTablesInternal(Consumer<Table> cacheTable, String dat
             return glueTables.stream()
                     .map(table -> new TableInfo(
                             new SchemaTableName(databaseName, table.name()),
-                            TableInfo.ExtendedRelationType.fromTableTypeAndComment(table.tableType(), table.parameters().get(TABLE_COMMENT))))
+                            TableInfo.ExtendedRelationType.fromTableTypeAndComment(
+                                    table.tableType() == null ? TABLE.name() : table.tableType(),
```

Comment:
nit: We could use `firstNonNull` method.

Reply:
`GlueConverter#fromGlueTable` sets `EXTERNAL_TABLE` by default. Why do we use the different value here?